string_utils.test.js

```diff
@@ -0,0 +1,66 @@
+/*jslint node: true */
+"use strict";
+
+var StringUtils = require('./string_utils');
+var STRING_JOIN_CHAR = StringUtils.STRING_JOIN_CHAR;
+var getSourceString = StringUtils.getSourceString;
+
+function runTest(name, expectation, func) {
```

Comment:
This will dropped once https://github.com/byteball/byteballcore/pull/48 is merged and the repo has access to yarn+ava.